pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -37,8 +37,8 @@ const (
 	canaryTaskSetARNKeyName = "canary-taskset-arn"
 	canaryServiceKeyName    = "canary-service-object"
 	// Stage metadata keys.
-	trafficRoutePrimaryMetadataKey = "primary-traffic"
-	trafficRouteCanaryMetadataKey  = "canary-traffic"
+	trafficRoutePrimaryMetadataKey = "primary-percentage"
+	trafficRouteCanaryMetadataKey  = "canary-percentage"
```

Comment:
Change the key name so that we can reuse the web view defined key name
ref: https://github.com/pipe-cd/pipe/pull/2082/files#diff-f1edd0e8d855531a0d162b13744ae393436bcf4a6f2e9b795254aaa333c44a14R62-R67